frontend/src/pages/meditation/meditation-entries-list/meditation-entry/styles.module.scss

```
@@ -0,0 +1,69 @@
+.track {
+  display: flex;
+  justify-content: space-between;
+  max-width: 451px;
+  max-height: 177px;
+  margin: 0 auto;
+  border-radius: 30px;
+  box-shadow: 0 4px 23px 0 var(--violet-100);
+}
+
+.background-image {
+  width: 185px;
+  height: 100%;
+  margin-right: -30px;
+  border-radius: 30px 0 0 30px;
+}
+
+.content {
+  display: flex;
+  flex-basis: 286px;
+  align-items: center;
+  padding: 48px 23px 44px 29px;
+  background-color: var(--white);
+  border-radius: 30px;
+}
+
+.info {
+  margin-right: 35px;
+  line-height: 1.25;
+}
+
+.title {
+  margin: 0;
+  font-weight: 600;
+  font-size: 16px;
+}
+
+.duration {
+  display: inline-block;
+  margin-top: 8px;
+  padding: 6px 12px;
+  color: var(--blue-400);
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 1.2;
+  text-align: center;
+  background-color: var(--blue-alpha-30);
+  border-radius: 14px;
+}
+
+.play-button {
+  display: inline-block;
+  min-width: 48px;
+  min-height: 48px;
+}
+
+.play-button:hover {
+  transform: scale(1.1);
+}
+
+@media (width < 600px) {
```

try to use already existing breakpoints
```suggestion
@media (width < 767px) {
```

---

Done